### PR TITLE
Ported the Project to React and Express

### PR DESCRIPTION
**Pull request by rahulpoonia29**
Ported the main app to React JS frontend and Express backend. Completed the issue https://github.com/Satyam1923/Spring-Music-Player/issues/11

### CODE REVIEW

**Comment by rahulpoonia29** [comment]
@Satyam1923 Please review the pull request.

File: frontend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Logs
```
Comment (Satyam1923) [inline frontend/.gitignore:1]: Don't push .git ignore file.

File: backend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
```
Comment (Satyam1923) [inline backend/.gitignore:2]: don't push git ignore files

File: App.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
+// App.js
```
Comment (Satyam1923) [inline App.js]: Shift these files to frontend folder

File: frontend/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+				<ul className="details">
+					<li className="name">
+						{data && data.length > 0 && data[currplaying].name}
+					</li>
+					<li className="author">
+						{data && data[currplaying].artist}{" "}
+						{data && data[currplaying].year}
+					</li>
+				</ul>
+				<AudioPlayer
```
Comment (Satyam1923) [inline frontend/src/App.jsx:68]: Dont show player untill we search for songs.

File: frontend/.gitignore (hunk 1 above)
Comment (rahulpoonia29) [reply frontend/.gitignore:1]: Both the folders are different and require different package.json

File: backend/.gitignore (hunk 2 above)
Comment (rahulpoonia29) [reply backend/.gitignore:2]: Both the folders are different and require different package.json

File: frontend/src/App.jsx [hunk 5]
```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from "react";
+import AudioPlayer from "react-h5-audio-player";
+import "react-h5-audio-player/lib/styles.css";
+import "./App.css";
+import axios from "axios";
+
+const App = () => {
+	const [data, setData] = useState(null);
```
Comment (Satyam1923) [inline frontend/src/App.jsx:8]: create a separate function for this and call under app.
It would be simpler to read.

File: frontend/README.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# React + Vite
```
Comment (Satyam1923) [inline frontend/README.md:1]: remove this file.

**Pull request merged**